puppet/zulip/templates/supervisor/zulip.conf.template.erb

```diff
@@ -26,7 +26,7 @@ directory=/home/zulip/deployments/current/
 <% if @tornado_multiprocess -%>
 [program:zulip-tornado]
 command=env PYTHONUNBUFFERED=1 /home/zulip/deployments/current/manage.py runtornado 127.0.0.1:98%(process_num)02d
-process_name=port-98%(process_num)02d
+process_name=zulip-tornado-port-98%(process_num)02d
```

Comment:
I think this change requires updates to this too:
```
tabbott@coset:~/zulip$ git grep zulip-tornado:port
scripts/restart-server:        subprocess.check_call(["supervisorctl", "restart", f"zulip-tornado:port-{p}"])
```

Reply:
Whoops.  Thanks for the catch!